### PR TITLE
Update docs to remove bluemix refs.

### PR DESCRIPTION
**Pull request by rabbah**
This is a doc update to remove references to openwhisk.ng.bluemix.net from the docs.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [x] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: docs/rest_api.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ You can also retrieve these credentials using the CLI running `wsk property get
 The following is an example that uses the [cURL](https://curl.haxx.se) command tool to get the list of all packages in the `whisk.system` namespace:
 
 ```bash
-curl -u USERNAME:PASSWORD https://openwhisk.ng.bluemix.net/api/v1/namespaces/whisk.system/packages
+curl -u USERNAME:PASSWORD https://APIHOST/api/v1/namespaces/whisk.system/packages
```
Comment (markusthoemmes) [inline docs/rest_api.md]:
```suggestion
curl -u USERNAME:PASSWORD https://$APIHOST/api/v1/namespaces/whisk.system/packages
```

File: docs/rest_api.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ curl -u USERNAME:PASSWORD https://openwhisk.ng.bluemix.net/api/v1/namespaces/whi
 ]
 ```
 
-In this example the authentication was passed using the `-u` flag, you can pass this value also as part of the URL as `https://$AUTH@{APIHOST}`
+In this example the authentication was passed using the `-u` flag, you can pass this value also as part of the URL as `https://USERNAME:PASSWORD@APIHOST`.
```
Comment (markusthoemmes) [inline docs/rest_api.md]:
```suggestion
In this example the authentication was passed using the `-u` flag, you can pass this value also as part of the URL as `https://$USERNAME:$PASSWORD@$APIHOST`.
```

File: docs/rest_api.md [hunk 3]
```diff
@@ -91,15 +91,15 @@ The OpenWhisk API supports request-response calls from web clients. OpenWhisk re
...
+**Note:** HTTP request and response bodies will only be truncated if they exceed 1000 bytes.
 
 Let's try getting the namespace value for the current user.
 ```
 wsk namespace list -v
 ```
 ```
 REQUEST:
-[GET]  https://openwhisk.ng.bluemix.net/api/v1/namespaces
+[GET]  https://{APIHOST}/api/v1/namespaces
```
Comment (markusthoemmes) [inline docs/rest_api.md]:
```suggestion
[GET]  https://${APIHOST}/api/v1/namespaces
```

File: docs/rest_api.md [hunk 4]
```diff
@@ -121,20 +121,22 @@ Response body received:
 ["john@example.com_dev"]
 ```
 
-As you can see you the printed information provides the properties of the HTTP request, it performs a HTTP method `GET` on the URL `https://openwhisk.ng.bluemix.net/api/v1/namespaces` using a User-Agent header `OpenWhisk-CLI/1.0 (<CLI-Build-version>)` and Basic Authorization header `Basic XXXYYYY`.
+As you can see you the printed information provides the properties of the HTTP request, it performs a HTTP method `GET` on the URL `https://{APIHOST}/api/v1/namespaces` using a User-Agent header `OpenWhisk-CLI/1.0 (<CLI-Build-version>)` and Basic Authorization header `Basic XXXYYYY`.
```
Comment (markusthoemmes) [inline docs/rest_api.md]:
```suggestion
As you can see you the printed information provides the properties of the HTTP request, it performs a HTTP method `GET` on the URL `https://${APIHOST}/api/v1/namespaces` using a User-Agent header `OpenWhisk-CLI/1.0 (<CLI-Build-version>)` and Basic Authorization header `Basic XXXYYYY`.
```

File: docs/rest_api.md (hunk 1 above)
Comment (rabbah) [reply docs/rest_api.md]: thanks - i used APIHOST without the $ explicitly here to match the USERNAME:PASSWORD --- and added a note later before the curl examples about $APIHOST.

i can change to {APIHOST} perhaps - the doc uses that earlier (and similarly replace U:P).

File: docs/rest_api.md [hunk 5]
```diff
@@ -34,7 +34,7 @@ These are the collection endpoints:
 - `https://{APIHOST}/api/v1/namespaces/{namespace}/packages`
 - `https://{APIHOST}/api/v1/namespaces/{namespace}/activations`
 
-The `{APIHOST}` is the OpenWhisk API hostname (for example, openwhisk.ng.bluemix.net, 172.17.0.1, and so on).
+The `{APIHOST}` is the OpenWhisk API hostname (for example, localhost, 172.17.0.1, and so on).
```
Comment (rabbah) [inline docs/rest_api.md]: @markusthoemmes notice here how APIHOST is introduced -- i preserved this in the early section.
Maybe this just confusing and I should make them all $APIHOST instead. Thoughts?

**Pull request merged**